src/st/no0/5BDCC.c

```diff
@@ -1,8 +1,134 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 #include "common.h"
-#include "game.h"
+#include "no0.h"
 
-INCLUDE_ASM("st/no0/nonmatchings/5BDCC", func_us_801DBDCC);
+#define VASE self + 1
+#define BEAKER self + 2
+#define WALKING_TABLE self + 3
+
+extern u16 D_us_80180C14[];
+extern s16 D_us_801825CC[];
+extern u16 D_us_801825D4[];
+extern u8 D_us_801825F4[];
+
+typedef enum {
+    /* 0 */ OUIJA_TABLE_INIT,
+    /* 1 */ OUIJA_TABLE_INIT_SUBENTITIES,
+    /* 2 */ OUIJA_TABLE_COME_ALIVE,
+    /* 3 */ OUIJA_TABLE_ATTACK,
+    /* 4 */ OUIJA_TABLE_UNUSED,
+    /* 255 */ OUIJA_TABLE_DEBUG = 255,
+} OuijaTableSteps;
+
+// Main Ouija Table entity
+void EntityOuijaTable(Entity* self) {
+    Entity* entity;
```

Comment:
perhaps name it "other" or something, just so it has its own name instead of being generically "entity".